test/_init_tests.js

```diff
@@ -188,6 +188,11 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
```

Comment:
You'll need to move this above the other, then. You'll never get to this line now.